Makefile

```diff
@@ -496,8 +496,6 @@ create-workload-cluster: $(ENVSUBST)
 		elif [[ "${CLUSTER_TEMPLATE}" == *windows* ]]; then \
 			echo "windows being applied" \
 			kubectl --kubeconfig=./kubeconfig apply -f templates/addons/windows; \
-		else \
```

Comment:
what about windows and ipv6 test templates? are those using cluster resource sets too?

Reply:
Looks like they both are. Looks like I can completely remove the if statement?

Reply:
If they're no longer needed let's remove the entire if statement.

I wonder if the conformance tests have the CRS as well though, running one below to confirm